src/main/java/net/datafaker/annotations/EmbeddedFake.java

```diff
@@ -0,0 +1,10 @@
+package net.datafaker.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface EmbeddedFake { }
```

Comment:
Would it be possible to add a small bit of documentation on this? It's not really clear to me what an EmbeddedFake is.